Kubernetes/omsagent.yaml

```diff
@@ -482,4 +523,67 @@ spec:
       configMap:
         name: container-azm-ms-agentconfig
         optional: true
-
+    - name: azurefile-pv
+      persistentVolumeClaim:
+        claimName: azurefile
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: replicaset-service
+  namespace: kube-system
+spec:
+  selector:
+    rsName: "omsagent-rs"
+  ports:
+  - protocol: TCP
+    port: 25235
+    targetPort: in-rs-tcp
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: azurefile
+provisioner: kubernetes.io/azure-file
+mountOptions:
+  - dir_mode=0777
+  - file_mode=0777
+  - uid=1000
+  - gid=1000
+parameters:
+  skuName: Standard_LRS
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Hope all these things go away as we move away from azure file auto provisioning.